test/MvNormal/mean_test.jl

```diff
@@ -8,4 +8,17 @@
         μ2, Σ2 = rand(rng, n), generate_random_posdef_matrix(rng, n)
         central_limit_theorem_test(ClosedFormExpectation(), Logpdf(MvNormal(μ1, Σ1)), MvNormal(μ2, Σ2), 10^6)
     end
-end
\ No newline at end of file
+end
+
+@testitem "mean(::ClosedFormExpectation, p::Logpdf{LinearLogGamma}, q::MultivariateNormalDistributionsFamily)" begin
+    include("mvnormal_utils.jl")
+
+    rng = StableRNG(123)
+    for _ in 1:10
+        n = rand(rng, 2:5)
+        μ, Σ = rand(rng, n), generate_random_posdef_matrix(rng, n)
+        α, β = rand(rng)*10, 1+rand(rng)
```

Comment:
Because we have an exponential transformation when we are taking expectation here numbers could be really big, if `\alpha` and `\beta` ratio is too big or too small, so just to avoid numerical instability problems I constrained them to some reasonable values.